modules/ocf_mirrors/files/healthcheck

```diff
@@ -19,9 +20,29 @@ import requests
 Mirror = namedtuple('Mirror', ['url', 'updated_at'])
 
 
-def get_updated(mirror_url):
-    """Find the time the host was last updated.
+def update_func(healthcheck):
+    if healthcheck == 'debian':
+        return get_updated_debian
+    elif healthcheck == 'ts':
+        return get_updated_ts
+    else:
+        return get_updated_debian
```

Comment:
This seems a bit redundant, maybe raise an error if the type is not one of the accepted ones? That seems like it would be a problem if it does actually happen.